subcommands/keys/tuf_updates_review.go

```diff
@@ -56,9 +61,37 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 		} else {
 			rootToShow = newCiRoot
 		}
-		bytes, err := subcommands.MarshalIndent(rootToShow, "", "  ")
-		subcommands.DieNotNil(err)
-		fmt.Println(string(bytes))
+
+		if showRaw {
+			bytes, err := subcommands.MarshalIndent(rootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			fmt.Println(string(bytes))
+		} else {
+			var baseRootToShow *client.AtsTufRoot
+			if showProd {
+				if updates.Current.ProdRoot != "" {
+					subcommands.DieNotNil(
+						json.Unmarshal([]byte(updates.Current.ProdRoot), &baseRootToShow),
+						"Current prod root",
+					)
+				} else {
+					// First rotation, old prod root equals old CI root
+					baseRootToShow = oldCiRoot
+				}
+			} else {
+				baseRootToShow = oldCiRoot
+			}
+
+			before, err := subcommands.MarshalIndent(baseRootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			after, err := subcommands.MarshalIndent(rootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			diff := godiff.Strings(
+				strings.Split(string(before), "\n"),
+				strings.Split(string(after), "\n"),
+			)
+			fmt.Println(strings.Join(diff, "\n"))
```

Comment:
The output is a little weird. It shows *everything*. I guess I was expecting classic unified diff where you only include a few context lines around the change. Still probably handy enough that someone will use it.

Reply:
I was trying to find a Golang library which does it exactly like the diff utility; but it was only a part of the `golang.org/x/tools/internal/lsp/diff` which is long gone. So, maybe, I should call the `diff` commandline tool instead, I was just not sure if it is present on every system.

At the end, I like how this minimalistic tool gives the full context; the root.json is not that large.
I could write a small function which filters lines far away from the diff. I just wanted to keep the scope manageable.

Reply:
let's not overthink it. this is good enough.